### PR TITLE
Just new version

### PR DESCRIPTION
**Pull request by myfoxtail**

### CODE REVIEW

File: just-http-layer/Plugin.js [hunk 1]
```diff
@@ -18,14 +18,20 @@ classes.Plugin = {
         return function (success, fail) {
             app.post('/api/plugins/save', app.parser(function (site, data, cb, user, res, req) {
 
+                console.log('asasdasdas');
+
```
Comment (okneigres) [inline just-http-layer/Plugin.js]: console.lob

File: just-http-layer/Plugin.js [hunk 2]
```diff
@@ -18,14 +18,20 @@ classes.Plugin = {
...
                 if (!user) {
                     res.status(403);
                     cb(['Authorization required']);
                     return;
                 }
 
+                console.log(123);
```
Comment (okneigres) [inline just-http-layer/Plugin.js]: console.log

File: just-jsapi-layer/just.extremeprog.angular.js [hunk 3]
```diff
@@ -68,7 +70,7 @@ angular.module('mongoSitesApi', [])
                     }
                 }
             }
-        })(mongoSitesApi, res);
+        })(Just, res);
```
Comment (okneigres) [inline just-jsapi-layer/just.extremeprog.angular.js]: mixed case. use lower

File: just-jsapi-layer/just.extremeprog.js [hunk 4]
```diff
@@ -1,10 +1,11 @@
...
- * @license MongoSitesAPI v0.7.0
- * (c) 2015-2016 extremeprog.com https://github.com/extremeprog-com/mongo-sites-api
+ * @license Just v0.7.0
+ *
+ * (c) 2015-2016 extremeprog.com https://github.com/extremeprog-com/just
  * License: MIT
  */
 
-mongoSitesApi = (function() {
+Just = (function() {
```
Comment (okneigres) [inline just-jsapi-layer/just.extremeprog.js]: use lowercase

**Review by okneigres — COMMENTED** [review]
Pls rename "Just" object to "just" and remove console.logs

**Comment by myfoxtail** [comment]
@okneigres fixed.

**Review by okneigres — APPROVED** [review]

**Pull request merged**